### PR TITLE
HAWQ-1310. Reformat resource_negotiator().

### PR DESCRIPTION
**Pull request by amyrazz44**
@jiny2 @linwen pls feel free to review this pull request, thank you.

### CODE REVIEW

**Comment by jiny2** [comment]
+1 LGTM. Thanks for fixing this so quickly.

**Comment by stanlyxiang** [comment]
+1

File: src/backend/optimizer/plan/planner.c [hunk 1]
```diff
@@ -438,256 +438,224 @@ PlannedStmt *refineCachedPlan(PlannedStmt * plannedstmt,
...
 
-	int optimizer_segments_saved_value = optimizer_segments;
+    int optimizer_segments_saved_value = optimizer_segments;
 
-	PG_TRY();
-	{
-    if (resourceNegotiateDone)
-    {
+    PG_TRY();
+        {
```
Comment (linwen) [inline src/backend/optimizer/plan/planner.c]: Indent from here doesn't align. Please fix.

File: src/backend/optimizer/plan/planner.c [hunk 2]
```diff
@@ -438,256 +438,224 @@ PlannedStmt *refineCachedPlan(PlannedStmt * plannedstmt,
...
+     * us some sense of the complexity of the query, and allocate the appropriate
+     * resource to run this query. After gaining the resource, we can perform the
+     * actual optimization.
+     */
+    increase_planning_depth();
+
+    plannerLevel++;
+    if (!resourceNegotiateDone) {
+        PG_TRY();
+            {
```
Comment (linwen) [inline src/backend/optimizer/plan/planner.c]: Indent here doesn't align. Please fix.

**Comment by jinyismilodon** [comment]
This has been delivered last week per my understanding, please close it @amyrazz44 . Thanks.

**Comment by jiny2** [comment]
Please close this pr as per my understanding it has been delivered last week.

**Pull request closed without merging**